src/index.html

```diff
@@ -7,6 +7,50 @@
   <title>基本コーディング</title>
 </head>
 <body>
+  <header class="l-header">
+    <nav class="l-header__nav">
+      <div class="l-header__pc-items">
+        <ul class="l-header__lists">
+          <li class="l-header__list"><a href="">ギズジョブ医療事務とは</a></li>
+          <li class="l-header__list"><a class="is-search" href="">こだわり検索</a></li>
+          <li class="l-header__list"><a class="is-book" href="">ギズジョブ医療事務コラム</a></li>
+        </ul>
```

Comment:
[nits]

`list` 自体が複数のことを表すと思うので、以下のような記述が良いかなと思ったのですが、いかがでしょう？

```suggestion
        <ul class="l-header__items">
          <li class="l-header__item"><a href="">ギズジョブ医療事務とは</a></li>
          <li class="l-header__item"><a class="is-search" href="">こだわり検索</a></li>
          <li class="l-header__item"><a class="is-book" href="">ギズジョブ医療事務コラム</a></li>
        </ul>
```

Reply:
itemに修正しました。
11c4359d1f430e7d7de5d1d0d5289e6ac5c477b7